src/book.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'item'
+class Book < Item
```

Comment:
- [ ] All public methods in all classes should have unit tests. please add tests for the rest of the classes.